news/tasks.py

```diff
@@ -491,7 +487,7 @@ def send_transactional_messages(data, transactionals):
             lang_code = 'en'
 
         msg_id = mogrify_message_id(tm.vendor_id, lang_code, 'H')
-        send_message.delay(msg_id, email, email, 'H')
+        send_message.delay(msg_id, email, email)
```

Comment:
nit: `format=None` not used anymore 
https://github.com/pmac/basket/blob/77d73ab592f974b2576c66840c4eb1b0b36fa430/news/tasks.py#L505